docs/component/identity-wallet.md

```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
 sidebar_label: Identity Mobile App
 ---
 
-Overview
+OA provides it's own mobile application to view, store and share Verifiable Document and Transferable Records.
 
-## Header
+## Installation
+
+For the moment, the application is not available on Apple store or Android store. You will need to run it through [Expo](https://expo.io/).
+
+1. Download Expo [![Expo on apple store](/docs/component/identity-wallet/apple-store.svg)](https://apps.apple.com/fr/app/expo-client/id982107779 "Open expo on apple store") [![Expo on apple store](/docs/component/identity-wallet/android-store.svg "Open expo on android store")](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en)
+1. Open Expo and download Identity Wallet app by scanning the following QR Code
+
+<img src="/docs/component/identity-wallet/identity-wallet-qr-code.png" alt="identity wallet qr code" title="scan the qr code with expo" width="200"/>
```

Comment:
confuse from here. Can not see any option to scan the QR code in Expo Client app.

Reply:
Hmmm if I open the app the first thing displayed is "scan qr code" ?

Reply:
I already had expo client app in my mobile. but can not see the option to scan any where. only  the screen shown in the app store and its weird dont even see the plus icon in my app on the top.

Reply:
Iphone / Android ?

Print screen of what you have ?

Reply:
ok will keep like this for the moment, until Raymond release on app store =)